internal/locate/region_cache.go

```diff
@@ -1603,17 +1603,17 @@ func (c *RegionCache) PDClient() pd.Client {
 	return c.pdClient
 }
 
-// GetTiFlashStoreAddrs returns addresses of all tiflash nodes.
-func (c *RegionCache) GetTiFlashStoreAddrs() []string {
+// GetTiFlashStores returns the information of all tiflash nodes.
+func (c *RegionCache) GetTiFlashStores() []*Store {
```

Comment:
I wonder if there will be some data race if we access `Store` later without acquiring locks.

Reply:
Yeah. But I checked every store access is protected by rwlock, why questioning that?

Reply:
okay.